packages/discord.js-utilities/src/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from '@sapphire/discord-utilities';
+export * from './lib/type-guards';
```

Comment:
You forgot to export your own PaginatedMessage stuff here

Reply:
Yeah, just now seeing that, thanks.